tracpro/baseline/views.py

```diff
@@ -73,8 +75,17 @@ def derive_queryset(self, **kwargs):
         def get_context_data(self, **kwargs):
             context = super(BaselineTermCRUDL.Read, self).get_context_data(**kwargs)
 
-            baseline_dict, baseline_dates = self.object.get_baseline(self.request.data_regions)
-            follow_ups, dates = self.object.get_follow_up(self.request.data_regions)
+            # Get the region from the region filter drop-down, if it was selected
+            region = int(self.request.GET.get('region', 0))
```

Comment:
We should surround this with `try... except...` just in case a bad value is passed in.